crates/aptos-rosetta/src/types/requests.rs

```diff
@@ -201,22 +197,21 @@ pub struct ConstructionMetadataResponse {
     /// Metadata that will be passed to Payloads to create a transaction
     pub metadata: ConstructionMetadata,
     /// A suggested gas fee based on the current state of the network
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub suggested_fee: Option<Vec<Amount>>,
+    pub suggested_fee: Vec<Amount>,
 }
 
 /// Metadata required to construct a transaction
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct ConstructionMetadata {
     /// Sequence number of the sending account
-    pub sequence_number: u64,
+    pub sequence_number: U64,
     /// Maximum gas willing to pay for the transaction
-    pub max_gas: u64,
+    pub max_gas_amount: U64,
     /// Multiplier e.g. how much each unit of gas is worth in the native coin
-    pub gas_price_per_unit: u64,
-    /// Expiry time of the transaction in unix epoch seconds
+    pub gas_price_per_unit: U64,
+    /// Unix timestamp of expiry time, defaults to 30 seconds from the payload request
```

Comment:
Nit: I wouldn't mention the default by value here, it could change. Same for the other comment above like this.

Reply:
You can't really change defaults here because it changes functionality at this point